fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/AbstractCumulativeLoanScheduleGenerator.java

```diff
@@ -2687,6 +2689,8 @@ private Money fetchEarlyPaidAmount(final Money principalPortion, final Money pri
                 // until this
                 // amount becomes zero
                 break;
+                case ADJUST_LAST_UNPAID_PERIOD:
```

Comment:
For cumulative loan this option shall not be available!

Reply:
Done! removed